cabal-testsuite/PackageTests/ListBin/Script/cabal.test.hs

```diff
@@ -0,0 +1,8 @@
+import Test.Cabal.Prelude
+
+main = cabalTest . void $ do
+    res <- cabal' "list-bin" ["script.hs"]
+
+    env      <- getTestEnv
+    cacheDir <- getScriptCacheDirectory $ testCurrentDir env </> "script.hs"
+    assertOutputContains cacheDir res
```

Comment:
@bacchanalia : I don't understand how this `assertOutputContains` succeeds with above output (that does not contain something resembling a `cacheDir`).  But apparently it does.  How?